src/components/Orders.js

```diff
@@ -39,10 +50,12 @@ class Orders extends React.Component {
         ItemQuantity: item[6],
         PickupStatus: (
           <Checkbox
-            key={item[3]}
-            pickUpStatus={item[7]}
+            key={item[3] + item[5]}
```

Comment:
I made this change but just fyi if the key is just order number it is not unique for every row